### DISCUSSION TITLE
"vercel dev" shows NODE_ENV as "development" in React App, but "production" in serverless functions

### DISCUSSION THREAD

**Discussion by vebits** [vercel/vercel#4765] [Help]
I have a create react app and a `api` folder with several serverless functions. When running `vc dev`, `process.env.NODE_ENV` is `development` on frontend, but in a serverless function in the `NODE_ENV` is set to `production`.

I have two `package.json`, one in the `src` frontend folder and one in the `api` folder.

How can I get `NODE_ENV = development` in serverless functions when running `vc dev`?

**Comment by nickdandakis** [comment]
I fixed this by explicitly setting `NODE_ENV=development` in `.env`.

**Reply by Fxlr8** [reply]
did not help in my case
